api/Notification.json

```diff
@@ -5,18 +5,13 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Notification",
         "spec_url": "https://notifications.spec.whatwg.org/#api",
         "support": {
-          "chrome": [
-            {
-              "version_added": "22",
-              "notes": "Before Chrome 22, the support for notification followed an old prefixed version of the specification and used the <code>navigator.webkitNotifications</code> object to instantiate a new notification. Before Chrome 32, <code>Notification.permission</code> was not supported. Before Chrome 42, service worker additions were not supported. Starting in Chrome 49, notifications do not work in incognito mode."
-            },
-            {
-              "prefix": "webkit",
-              "version_added": "5"
-            }
-          ],
+          "chrome": {
+            "version_added": "20",
+            "notes": "Before Chrome 42, service worker additions were not supported. Starting in Chrome 49, notifications do not work in incognito mode."
```

Comment:
OK, these are clearly two distinct notes now. Can we split these notes into an array?

* From Chrome 49…
* Before Chrome 42…